sdk/script/management-sdk.ts

```diff
@@ -257,8 +251,15 @@ class AccountManager {
         });
     }
 
-    public promote(appName: string, sourceDeploymentName: string, destDeploymentName: string): Promise<void> {
-        return this.post(urlEncode `/apps/${appName}/deployments/${sourceDeploymentName}/promote/${destDeploymentName}`, /*requestBody=*/ null, /*expectResponseBody=*/ false)
+    public patchRelease(appName: string, deploymentName: string, packageInfo: PackageInfo): Promise<void> {
```

Comment:
I think we should still expose `label` as a "positional argument", and rename `packageInfo` to `updatedMetadata`.